src/controllers/BoardController.ts

```diff
@@ -24,4 +34,47 @@ export class BoardController {
 
         return res.status(statusCode.CREATED).send(util.success(statusCode.OK, message.READ_BAORD_LIST_SUCCESS, boards))
     }
-}
\ No newline at end of file
+
+    @UseBefore(verifyAccessToken)
+    @HttpCode(200)
```

Comment:
```suggestion
    @HttpCode(201)
```

게시물 생성로직이니 201번을 반환해주는게 좋은 것 같습니다